server.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
```

Comment:
Consider removing this file and all the related packages.
We would not need it at this point.

Reply:
Updated as per the comment